src/containers/Editor/BottomBar.tsx

```diff
@@ -115,6 +116,8 @@ export const BottomBar = () => {
     }
   }, [getJson, hasChanges, query.json, replace, setHasChanges, setVisible, user]);
 
+  useHotkeysforCtrlAndMeta(83, handleSaveJson);
```

Comment:
move it to the line 77

Reply:
Hi @AykutSarac, i will do this tonight, is there any other point apart from this i will update.

Reply:
Hi @AykutSarac, i will do this tonight, is there any other point apart from this i will update.

Reply:
I can't move this line to 77 because it needs to define **handleSaveJson()** function before